x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/home.spec.ts

```diff
@@ -66,17 +60,11 @@ describe.skip('Home page', () => {
 
   describe('navigations', () => {
     it('navigates to service overview page with transaction type', () => {
-      apisToIntercept.map(({ endpoint, name }) => {
-        cy.intercept('GET', endpoint).as(name);
-      });
-
       cy.visit(serviceInventoryHref);
 
       cy.contains('Services');
+      cy.contains('opbeans-rum').wait(3000).click({ force: true });
```

Comment:
Waiting on DOM elements is discouraged but `click` causes the test to be flaky. As we've seen this quite a few times so far, we might benefit by using [cypress-pipe](https://github.com/NicholasBoll/cypress-pipe) (also used by [security](https://github.com/elastic/kibana/pull/91550))